vslib/VirtualSwitchSaiInterface.cpp

```diff
@@ -1298,6 +1298,12 @@ sai_status_t VirtualSwitchSaiInterface::bulkCreate(
 {
     SWSS_LOG_ENTER();
 
+    // create new real object IDs
+    for (uint32_t idx = 0; idx < object_count; idx++)
+    {
+        object_id[idx] = m_realObjectIdManager->allocateNewObjectId(object_type, switch_id);
+    }
+
```

Comment:
was that missing or this is already done in bulkCreate( generic in line 1310 ?

Reply:
@kcudnik it was missing, this part has been taken from single create method